src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -65,7 +66,8 @@
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlation_id=None, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
                         kube_config=None, kube_context=None, no_wait=False, tags=None, distribution='generic', infrastructure='generic',
                         disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="600", enable_private_link=None, private_link_scope_resource_id=None,
-                        distribution_version=None, azure_hybrid_benefit=None, skip_ssl_verification=False, yes=False, container_log_path=None):
+                        distribution_version=None, azure_hybrid_benefit=None, skip_ssl_verification=False, yes=False, container_log_path=None,
+                        enable_oidc_issuer=False, enable_workload_identity=False, self_hosted_issuer=""):
```

Comment:
To confirm, we want both to be independent of each other?

Reply:
by both do you mean oidc and workload identity? Yes they will be 2 separate flags!